examples/MySQL-client/CMakeLists.txt

```diff
@@ -9,6 +9,10 @@ project(MySQL-client-example)
 hunter_add_package(MySQL-client)
 find_package(MySQL-client REQUIRED)
 
+if(MSVC) #LIBCMT conflicts with MSVCR
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /NODEFAULTLIB:LIBCMT.lib")
```

Comment:
Not sure, I checked on my local build when I saw the issue at first and the lib was compiling correctly with the applications settings and was still giving the issue. I found a stackoverflow question that said this could also be caused when compiling a debug build with a release library, which I thought might be the problem so I tracked down why the debug builds were disable. I had thought that fixed the issue but in the end it did not. So I had to put that in to make sure the builds succeeded.

It was also odd as VC14 only considered this a warning and did not classify this issue as an error like all the previous versions.